File: 240206/구간 중 최대 합/max-sum-of-subarray.py
n,k=map(int,input().split())
num=list(map(int,input().split()))

ans=0
for i in range(n-k+1):
    val=0
    for j in range(i,i+k):
        val+=num[j]
    ans=max(ans,val)
print(ans)

File: 260109/구간 중 최대 합/max-sum-of-subarray.js
const fs = require("fs");
const input = fs.readFileSync(0).toString().trim().split('\n');

const [n, k] = input[0].split(' ').map(Number);
const arr = input[1].split(' ').map(Number);

// Please Write your code here.

let ans = 0;

for(let i=0; i<n-k+1; i++){
    let sum = 0;
    for(let j=i; j<i+k; j++){
        sum+=arr[j];
    }
    ans = Math.max(ans, sum);
}

console.log(ans)